src/common/latest/jest.config.js

```diff
@@ -1,5 +1,7 @@
 const path = require("path");
 
+const esModules = ["node-fetch", "data-uri-to-buffer", "fetch-blob", "formdata-polyfill"].join("|");
```

Comment:
this is needed to transform with babel the `node-fetch` dependency for jest to be able to run the tests without mocking the entire module